src/test/java/de/mediathekview/mserver/crawler/mdr/parser/MdrFilmXmlHandlerTest.java

```diff
@@ -0,0 +1,110 @@
+package de.mediathekview.mserver.crawler.mdr.parser;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import de.mediathekview.mlib.daten.Resolution;
+import de.mediathekview.mserver.testhelper.FileReader;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collection;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.xml.sax.SAXException;
+
+@RunWith(Parameterized.class)
+public class MdrFilmXmlHandlerTest {
+
+  private String xmlFile;
+  private String expectedTopic;
+  private String expectedTitle;
+  private String expectedDescription;
+  private LocalDateTime expectedTime;
+  private Duration expectedDuration;
+  private String expectedWebsite;
+  private String expectedUrlSmall;
+  private String expectedUrlNormal;
+  private String expectedUrlHd;
+  private String expectedSubtitle;
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(
+        new Object[][]{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)